reactive/k8s.py

```diff
@@ -53,13 +61,37 @@ def download_kubectl():
     status_set('maintenance', 'Kubernetes installed')
 
 
-def render_manifest(reldata):
-    data = {'connection_string': reldata.connection_string()}
-    render('master.json', 'files/manifests/master.json', data)
+def render_files(reldata):
```

Comment:
This is a candidate for extrapolating into a charm-helper for launching compose based apps. We should discuss this further. I like whats here though :+1: